conbench/config.py

```diff
@@ -25,6 +25,27 @@ class Config:
     # to recalculate history.
     DISTRIBUTION_COMMITS = int(os.environ.get("DISTRIBUTION_COMMITS", 100))
 
+    # The base URL (scheme, DNS name, path prefix) that regular HTTP clients
+    # are expected to use for reaching the HTTP server exposing the app.
+    # Depends on the deployment and cannot generally be determined by the app
+    # itself (requires human input). The default value is tailored to the Flask
+    # development HTTP server defaults (non-TLS, binds on 127.0.0.1, port
+    # 5000). Can be adjusted via the `--host` and `--port` command line flags
+    # when invoking `flask run`. Three interesting scenarios:
+    # - local dev setup with default listen address: no action needed
+    # - local dev setup with custom listen address: user should set meaningful
+    #   value
+    # - prod/staging setup: user should set meaningful value such as for
+    #   example https://conbench.ursa.dev/
+    #
+    # Currently used for populating the 'Servers' dropdown in the
+    # Swagger/OpenAPI docs website.
```

Comment:
💯 I know long comments are frequently looked down on as uncouth, but I love _love_ all the details here for our future selves – thank you!